Client/Pages/Repl.razor.cs

```diff
@@ -79,12 +84,16 @@ public async Task CompileAsync()
                 this.Diagnostics = result.Diagnostics.OrderByDescending(x => x.Severity).ThenBy(x => x.Code).ToList();
                 this.AreDiagnosticsShown = true;
             }
+            catch (Exception)
+            {
+                this.PageNotificationsComponent.AddNotification(NotificationType.Error, content: "Error while compiling the code.");
+            }
             finally
             {
                 this.Loading = false;
             }
 
-            if (result.AssemblyBytes != null && result.AssemblyBytes.Length > 0)
+            if (result?.AssemblyBytes != null && result.AssemblyBytes.Length > 0)
```

Comment:
we might merge this into
```suggestion
            if (result?.AssemblyBytes?.Length > 0)
```